riotfile.py

```diff
@@ -2685,6 +2685,16 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "anthropic": [latest, "~=0.28"],
             },
         ),
+        Venv(
+            name="google_generativeai",
+            command="pytest {cmdargs} tests/contrib/google_generativeai",
+            pys=select_pys(min_version="3.9"),
+            pkgs={
+                "pytest-asyncio": latest,
+                "google-generativeai": ["~=0.7.2"],
```

Comment:
nit: Do we not need `latest` here?

Reply:
0.7.2 is the latest version, wanted to avoid unnecessary double testing. I will instead replace this with latest